main/src/ca/uwaterloo/flix/api/lsp/Index.scala

```diff
@@ -62,6 +67,7 @@ object Index {
   */
 case class Index(m: Map[(String, Int), List[Entity]],
                  defUses: MultiMap[Symbol.DefnSym, SourceLocation],
+                 sigUses: MultiMap[Symbol.SigSym, SourceLocation], // MATT do we want classUses instead?
```

Comment:
No this is good! We may want both.

Imagine you have:

```
trait Eq[a] {
  def ==(...)
}
```

then you want to be able to right-click on both "Eq" and "==" and ask "Find all Usages".

Reply:
Presumably right-clicking on == finds all uses of the actual function. Whereas right-clicking on Eq finds all mentions of the type.

Reply:
These are different places in the source code.